docs/griptape-framework/engines/src/eval_engines_1.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from griptape.engines import EvalEngine
+
+engine = EvalEngine(
+    criteria="Determine whether the actual output is factually correct based on the expected output.",
+    evaluation_params=[
+        EvalEngine.Param.INPUT,
+        EvalEngine.Param.EXPECTED_OUTPUT,
+        EvalEngine.Param.ACTUAL_OUTPUT,
+    ],
+)
+
+score, reason = engine.evaluate(
+    input="If you have a red house made of red bricks, a blue house made of blue bricks, what is a greenhouse made of?",
+    expected_output="Glass",
+    actual_output="Glass",
+)
```

Comment:
Without looking at the implementation (not hypothetical -- I haven't), it looks weird that I need to say which parameters I'm going to pass into a method in the constructor. Why can't this be implied given the parameters that I actually pass into evaluate?

Reply:
I think we can, updated the implementation.